src/builder/build_traits.rs

```diff
@@ -325,6 +325,8 @@ pub trait Dataflow: Container {
             NodeType::open_extensions(ops::CFG {
                 inputs: inputs.clone(),
                 outputs: output_types.clone(),
+                // TODO: Make this a parameter
```

Comment:
I'll do another PR to make these methods take `FunctionType` as an argument and taking the `extension_delta` from there